tests/cpp/test_multidevice_transformer.cpp

```diff
@@ -50,21 +57,9 @@ class DistributedTransformerTest
       .use_fusion_executor_cache = true,
       .skip_auto_scheduling = false,
       .cache_fusion_executor = false};
-  const int D;
-  static constexpr int B = 4;
-  static constexpr int E = 128;
-  static constexpr int H = 4;
-  static constexpr int S = 32;
-
- private:
-  // Note: `MoveSplitCat` and `AllocationDomain` preseg passes use ID model.
-  // `SdpaFwdOp` currently does not work with ID model since it requires all
-  // sibling outputs to have the same root domain.
-  //  This will be modified in a future PR.
-  preseg_passes::OptimizationPassGuard<preseg_passes::MoveSplitCatPass>
-      optimization_guard_;
-  preseg_passes::OptimizationPassGuard<preseg_passes::AllocationDomainPass>
-      allocation_order_guard_;
+
+ public:
+  const int64_t D; // number of devices
```

Comment:
```suggestion
  const int64_t num_devices_;
```

Reply:
I think I'll leave this as D, just for when Attention layer PR gets merged - there are a lot of a reshapes/slices, so using `D` makes it a bit easier to read: https://github.com/NVIDIA/Fuser/pull/2590/files#diff-4e10caf37fae9be5bfb0bc52891227c5ed5141656d0396d5c2a60e06dd1dd1ebR227